### DISCUSSION TITLE
Distant Horizons Compatibility!

### DISCUSSION THREAD

**Discussion by Wulfic** [IrisShaders/Iris#2230] [Ideas]
https://www.curseforge.com/minecraft/mc-mods/distant-horizons

Would love to see compatibility added for this mod as it brings the world draw distance to a whole new level using LoDs!

**Comment by tavaresgerson** [comment]
Nice suggestion bro!

**Comment by benjamin051000** [comment]
I'm gonna copy and paste the "official" documentation regarding DH compatibility FAQ in the Iris discord server so it's more easily discoverable here. They were posted 3/5/2024, 3/9/2024, 3/16/2024 respectively.

# Where is Distant Horizons compatibility?
Distant Horizons compatibility is currently in early testing. It is unstable, and requires knowledge of modding to use. It is recommended to wait until the final release.

You will not recieve support for installing early betas; you are expected to know how to do so.

> [!WARNING]
> These early betas may cause damage to your worlds/and or computer. You have been warned.

There are two ways to install the beta:
- If you already have a Fabric instance, you can use one of these two links to get the latest versions for 1.20.1 or 1.20.4. (These are not mod files, you need to unzip.)
https://github.com/IrisShaders/Iris-Installer-Files/releases/latest/download/Iris-Sodium-Beta-1.20.1.zip
https://github.com/IrisShaders/Iris-Installer-Files/releases/latest/download/Iris-Sodium-Beta-1.20.4.zip

- If you do not have a Fabric instance, go to https://irisshaders.dev/ and use the installer. 

---
# Checklist before reporting issues
- Do you have the right Iris version? It should be labeled 1.7.0.
- Do you have the right DH version? It should be at least 2.0.2.
- Do you have a compatible shader from ⁠(discord channel) #dh-shader-testing ?
- Are you using incompatible mods like Physics Mod?
- If chunks aren't loading/are loading incredibly slow, increase the "CPU Load" amount in DH settings.

---
# Tutorial on installing by TroubleChute:
https://youtu.be/pIRSjKz9RLg

**Comment by TLiG98** [comment]
For us on Fabric 1.19+ don't have compatibility with DH 2.1.0 since they need the Iris 1.7+
Would it be possible to add Iris 1.7 for 1.19.2 as well?

**Comment by neoshifter39** [comment]
I second the 1.7 for 1.19.X that TLiG98 mentioned.

I have a personal 1.19.2 pack that I REALLY wanted to use with DH, that a few of the mods I have are ONLY for 1.19.2 at latest...

**Reply by TLiG98** [reply]
Same. So having 1.7+ for 1.19.2, where I am as well, would be so fantastic!
